### PR TITLE
force expand_cron_seconds argument into string

### PR DESCRIPTION
**Pull request by keymone**
puppet 4 sends fixnums into this function, ensure it's a string before running `#split` on it.

### CODE REVIEW

**Comment by solarkennedy** [comment]
Strict vars on `::osfamily` failing? I'm not sure how that worked before.

Can you expand the travis matrix to include puppet 4?

**Comment by keymone** [comment]
also: i've copied human_time_to_seconds from monitoring check and dropped it from fixtures.

File: lib/puppet/parser/functions/human_time_to_seconds.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+module Puppet::Parser::Functions
+  newfunction(
+    :human_time_to_seconds,
```
Comment (solarkennedy) [inline lib/puppet/parser/functions/human_time_to_seconds.rb]: The problem here is that this file will "fight" with the monitoring_check one.

You will have to namespace this function, how about `cron_human_time_to_seconds`?

In general I'm ok with "vendoring" this function, but you will have to give it a new name.

File: .travis.yml [hunk 2]
```diff
@@ -3,12 +3,15 @@ language: ruby
...
+
 rvm:
   - 1.8.7
   - 1.9.3
   - 2.0.0
   - 2.1.0
-script: bundle exec rake test
+  - 2.2.0
+  - 2.3.0
 env:
```
Comment (solarkennedy) [inline .travis.yml]: This has turned into an *epic* travis.yml.

I personally don't mind too much, but dang.

**Comment by solarkennedy** [comment]
fix the name + ship

**Pull request merged**